config/deploy/staging.rb

```diff
@@ -0,0 +1,7 @@
+server 'sul-robots5-test.stanford.edu', user: 'lyberadmin', roles: %w{web app db}
+
+Capistrano::OneTimeKey.generate_one_time_key!
+
+set :deploy_environment, 'test'
```

Comment:
the robots use `test` for the ROBOT_ENVIRONMENT -- we don't have a true staging environment for the robots